### ISSUE TITLE
Isn't Monday the first day of the week for 80%?

### ISSUE THREAD

**Issue by alanmels** [backdrop/backdrop-issues#4928]
Filing this as a question and not as a bug.

Backdrop now installs with Sunday as the first day of the week, however considering the 80% principle that Backdrop community follows, this could be changed to Monday, because of the following reasons:

1. Is it me thinking so or indeed Monday is the first day of the week for most users?
2. According to [ISO_8601](https://en.wikipedia.org/wiki/ISO_8601):
>  [Www] is the week number prefixed by the letter W, from W01 through W53. [D] is the weekday number, from 1 through 7, beginning with Monday and ending with Sunday. 

3. Also https://en.wikipedia.org/wiki/ISO_week_date says:

> 
Weeks start with Monday. Each week's year is the Gregorian year in which the Thursday falls. The first week of the year, hence, always contains 4 January. ISO week year numbering therefore usually deviates by 1 from the Gregorian for some days close to 1 January. 

4. Wikipedia page for Monday on https://en.wikipedia.org/wiki/Monday states:

> Monday is the day of the week between Sunday and Tuesday.[1] According to the international standard ISO 8601 it is the first day of the week. In countries that adopt the "Sunday-first" convention, it is the second day of the week. The name of Monday is derived from Old English Mōnandæg and Middle English Monenday, originally a translation of Latin dies lunae "day of the Moon".

Off-topic: I never knew it's translated as the "day of the Moon".

**Comment by herbdool** [comment]
Yeah, I would think Monday as first day is more typical.

**Comment by BWPanda** [comment]
> Backdrop now installs with Sunday as the first day of the week

I see nothing to suggest that this wasn't always the case...

https://github.com/backdrop/backdrop/blame/ac819913fdb7aadcec9c7579fd33920327526792/core/modules/system/config/system.date.json#L3

https://github.com/backdrop/backdrop/blob/0035d2af3ac569de2788b53384d85d8ca7e9ba58/core/modules/views/includes/utility.inc#L438-L439

**Comment by klonos** [comment]
Thanks for opening this issue @alanmels 👍 

>Is it me thinking so or indeed Monday is the first day of the week for most users?

Yeah, I think that the norm is Monday as first day of the week. Perhaps because it's the first workday of the week in most (all?) countries.

I think that we should change it to be Monday for new sites, leave it as is in existing sites.

**Comment by findlabnet** [comment]
> first workday of the week in most (all?) countries.

I know at least one country where Sunday is first working day of the week. ;-)

**Comment by indigoxela** [comment]
Another Wikipedia article: https://en.wikipedia.org/wiki/Week

It uses this map: https://commons.wikimedia.org/wiki/File:First_Day_of_Week_World_Map.svg

It seems, that there exists no "80%" in this case. :wink:

**Comment by herbdool** [comment]
I learn something new. I didn't know there was so much variety. Plus I must have forgotten that North America treats Sunday as the first day. I guess it doesn't really affect my life so I just assumed it was Monday.

**Comment by klonos** [comment]
> I learn something new.

Same here. I've always assumed that Sunday has been chosen because of PHP 🤔 ...I guess that this needs to be a regional setting.

**Comment by alanmels** [comment]
> Another Wikipedia article: https://en.wikipedia.org/wiki/Week
> 
> It uses this map: https://commons.wikimedia.org/wiki/File:First_Day_of_Week_World_Map.svg
> 
> It seems, that there exists no "80%" in this case. wink

Very interesting finding, which makes me now think the default setting is just fine. However, then comes another question: Why with regard to some settings - for example, for locale, language settings - we strictly follow ISO standards, but in this case we don't follow it (https://en.wikipedia.org/wiki/ISO_8601)?!

**Comment by cptX** [comment]
Greece for sure has Monday as the first working day of the week, and in every installation I'm questioning why Sunday is the default!

**Comment by domaingood** [comment]
I prefer the admin can choose Sunday Monday Any other from the setting.

**Comment by alanmels** [comment]
> I prefer the admin can choose Sunday Monday Any other from the setting.

Admin did and always will have that feature. However, we are talking about slightly different thing here: what should be the default setting.

**Comment by alanmels** [comment]
As I said earlier, I've changed my opinion in the course of this discussion and at the moment think the current setting is just ok. However, we have to follow some kind of rule/principle when it comes to ISA standards - either we follow them strictly or not. If we let some historical/religion based practices with regard to some settings, then we can not be strictly following when it comes to other settings as, for example, locales.

**Comment by klonos** [comment]
off-topic: @cptX are you also from Greece or located there?

**Comment by cptX** [comment]
Yes @klonos I am located in Athens. I sent you greetings in another post, as I realized you are greek too.
